cypress/features/regression/experimental/numberInput.feature

```diff
@@ -81,7 +81,7 @@ Feature: Experimental Number Input component
 
   @positive
   Scenario Outline: Set label align to <labelAlign>
-    When I open default "Experimental Number Input" component in noIFrame with "numberInput" json from "experimental" using "<nameOfObject>" object name
+    When I open default "Experimental Number Input Test" component in noIFrame with "numberInput" json from "experimental" using "<nameOfObject>" object name
```

Comment:
If we will add `labelAlign` as a separate story in `mdx` we could delete this TC